tests/ui/version-flags/version-info-flags.rs

```diff
@@ -0,0 +1,11 @@
+//@ revisions: version verbose-version long-verbose-version
+//@[version] check-pass
```

Comment:
I just realized since check-pass isn't conditioned on the revision, it can just be extracted as unrevisioned directive

//@ check-pass

so we don't need to repeat it for each revision

Reply:
Neat! Applied.